### DISCUSSION TITLE
Bookings-toiminnon käännös?

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha-translations#28] [Yleinen]
23.11-versiossa on tulossa uusi Bookings-ominaisuus, jolla ilmeisesti pystyy asiakas bookkaamaan itselleen jonkin tietyn teoksen/niteen noudettavaksi tiettynä päivänä ja lainaan tiettyyn aikaväliin.

https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=29002

Miten tämä toiminnon suomentaisi?

Näitä ainakaan minusta ei voi käyttää, koska ovat jo muussa käytössä:
* varaus
* hyllyvaraus
* tilaus

Ehdotus:
* ennakkovaraus

**Comment by pknuuti** [comment]
Ehdotettu ennakkovaraus kuulostaa ihan järkevältä.

Päivi Knuutinen
kirjastovirkailija
Vaara-kirjastot
050 441 2247
***@***.***

Lähettäjä: Anneli Österman ***@***.***>
Lähetetty: 22. marraskuuta 2023 11:26
Vastaanottaja: KohaSuomi/Koha-translations ***@***.***>
Kopio: Subscribed ***@***.***>
Aihe: [KohaSuomi/Koha-translations] Bookings-toiminnon käännös? (Discussion #28)


23.11-versiossa on tulossa uusi Bookings-ominaisuus, jolla ilmeisesti pystyy asiakas bookkaamaan itselleen jonkin tietyn teoksen/niteen noudettavaksi tiettynä päivänä ja lainaan tiettyyn aikaväliin.

https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=29002

Miten tämä toiminnon suomentaisi?

Näitä ainakaan minusta ei voi käyttää, koska ovat jo muussa käytössä:

  *   varaus
  *   hyllyvaraus
  *   tilaus

Ehdotus:

  *   ennakkovaraus

-

**Comment by AnneliO** [comment]
Onko muilla mielipiteitä Bookings-termin käännöksestä?
